lib/strategy.js

```diff
@@ -10,9 +38,18 @@ function Strategy(options, verify) {
   passport.Strategy.call(this);
   this.name = 'lti';
   this._passReqToCallback = options.passReqToCallback;
-  if (options.createProvider) {
+
+  // If it's intended to check tool consumer's identity from database
+  // Dev will pass a function that handle it
+  if (typeof(options.createProvider) === "function") {
```

Comment:
`typeof` is a keyword, not a function.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof